server/routes/lists.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const router = express.Router();
+
+const User = require("../models/User");
+const List = require("../models/List");
+const Product = require("../models/Product");
+
+const verifyToken = require("../middleware/verify");
+
+// GET all lists of user
+router.get("/lists", verifyToken, function (req, res) {
+	User.findById(req.user.id).populate("my_lists").exec(function(err, foundUser) {
+		if(err){
+			console.log("error: User not found.");
```

Comment:
we probably want more than just console log. You should return a status code and a message so that errors can be displayed to the user